spec/classes/openvpn_config_spec.rb

```diff
@@ -4,9 +4,9 @@
   context 'on Debian based machines' do
     let(:facts) do
       {
-        osfamily: 'Debian',
-        operatingsystem: 'Debian',
-        operatingsystemrelease: '7',
+        os: { 'family' => 'Debian' },
```

Comment:
this should be migrated to rspec-puppet-facts, same applies for the other spec files.

Reply:
You're right, but the tag `needs-work` was not useless.
It did it.